requirements.txt

```diff
@@ -1,9 +1,9 @@
 click
+boto
```

Comment:
Why `boto`? It is also not listed in setup.py. 
When installing using `pip install` only packages listed in setup.py are installed.

Reply:
boto was a type-o. I deleted it.